### DISCUSSION TITLE
Stato automazione

### DISCUSSION THREAD

**Discussion by homepd** [indomus/forum#1388] [HUB: Home Assistant]
Ciao, la mia esigenza e' sapere se tutte le condizioni che permettono l'attivazione di un'automazione si stanno verificando. Provo a spiegarmi meglio, ho un'automazionre che come trigger ha la rilevazione di presenza ma le condizioni perche' si attivi  sono un certo livello di illuminazione e lo stato di una porta su open, ora vorrei che se queste condizioni sono su true visualizzare un bollino verde, se sono su false anche solo una delle 2 visualizzare un bollino rosso, esiste una soluzione che poetete suggerirmi? grazie

**Comment by snakuzzo** [comment]
Sì, puoi creare un [sensor template](https://www.home-assistant.io/integrations/template/#state-based-template-sensors) basato sullo stato degli stati delle entità che hai inserito nella condition.
Aiutati con i dev tools su lovelace

**Comment by homepd** [comment]
Grazie, comincio a darci un occhio, nel caso provo a romperti le scatole ;-)

**Reply by snakuzzo** [reply]
Cortesemente fai reply sotto l'ultima risposta...non creare nuovi thread sotto lo stesso topic.
Comunque non è che con il sensor template hai lo STATO dell'automazione...lo stato di un'automazione può essere `on` o `off`

**Reply by homepd** [reply]
Ciao, si certo, in effetti il titolo del post e' fuorviante, io voglio sapere non se l'automazione e' in On ma se le condizioni per cui il trigger faccia scattare l'On si stanno verificando.
Grazie

**Reply by snakuzzo** [reply]
no...il trigger è un conto e le condizioni da verificare un altro.
Inutile verificare in che stato sono le entità nella condition se l'automazione scatta e ti manda la notifica.
A mio avviso tutta questa cosa è inutile...può al limite interessarti creare un sensor template come già detto nella mia prima risposta.
Avrai un sensore (magari su lovelace) dove puoi sempre vedere in che stato è...indipendentemente dall'automazione

**Reply by homepd** [reply]
Grazie sono riuscito a farmi un sensor template in cui verifico che le condizioni necessarie a fare parttire l'automazione siano true. Cio' significa che una volta triggerata deve partire. A me non interessava capire lo stato dell'automazione se fosse in running o meno, volevo proprio tracciare il verificarsi delle condizioni. Forse non son in grado di spiegarne l'utilita' ma sono riuscito a creare il sensore. ti ringrazio ancora.
